internal/isoutil/isoutil.go

```diff
@@ -255,3 +256,7 @@ func (h *installerHandler) VolumeIdentifier() (string, error) {
 
 	return strings.TrimSpace(string(volumeId)), nil
 }
+
+func (h *installerHandler) CleanWorkDir() error {
+	return os.Remove(h.workDir)
```

Comment:
I think this needs to be `os.RemoveAll`

Otherwise we get this warning:
`time="2021-01-12T17:06:47Z" level=warning msg="Failed to clean isoHandler work dir" func="github.com/openshift/assisted-service/internal/isoeditor.(*rhcosEditor).CreateMinimalISOTemplate.func1" file="/go/src/github.com/openshift/origin/internal/isoeditor/rhcos.go:69" error="remove /tmp/baseiso823426943: directory not empty" go-id=306 request_id=`

Reply:
done